tpe/lib/src/Model.hh

```diff
@@ -93,6 +93,23 @@ class IGNITION_PHYSICS_TPELIB_VISIBLE Model : public Entity
     const math::Vector3d _linearVelocity,
     const math::Vector3d _angularVelocity);
 
+  /// \brief Remove a model entity by id
+  /// \param[in] _id Id of model entity to remove
+  public: bool RemoveModelById(std::size_t _id);
+
+  /// \brief Remove a link entity by id
+  /// \param[in] _id Id of link entity to remove
+  public: bool RemoveLinkById(std::size_t _id);
+
+  /// \brief Remove a child entity by id
+  /// \param[in] _id Id of child entity to remove
+  public: bool RemoveChildById(std::size_t _id) override;
```

Comment:
Nit: these are missing `\return`

Reply:
Fixed in https://github.com/ignitionrobotics/ign-physics/pull/230/commits/ea42d0cfe2f70e660419b3f206ed9800fecc5785